config/locales/forms/en.yml

```diff
@@ -3,6 +3,21 @@ en:
   forms:
     account_recovery_setup:
       legend: Select a secondary authentication option
+    backup_code:
+      info: Use backup codes if you cannot access your phone to sign in
+      instructions: You will be asked for a backup code <strong>each time you sign
+        in</strong> to your login.gov account.
+      new_codes: Generate new codes
+      new_info: When you generate new codes, you must download or print them. Your
+        old codes won't work anymore.
+      new_login_codes: Generate new login codes
+      print: Print
+      subinfo1: Each code may only be used once, once the 10th code has been used
```

Comment:
Why do we need to number the subinfo and subtitle keys? How many subtitles do we have?